templates/_init/flake.nix

```diff
@@ -3,5 +3,11 @@
 
   inputs.flox-floxpkgs.url = "github:flox/floxpkgs";
 
-  outputs = args @ {flox-floxpkgs, ...}: flox-floxpkgs.project args (_: {});
+  outputs = args @ {flox-floxpkgs, ...}:
+    flox-floxpkgs.project args (_: {
+      config.nixpkgs-config = {
+        allowUnfree = false;
+        allowedUnfreePackages = [];
```

Comment:
@garbas thoughts on this UX?

Reply:
what's the values to put into `allowedUnfreePackages`?

Reply:
Gave an example

Reply:
I guess we could consider putting this in `flox.nix` instead

Reply:
We've had a request for overlays too. That goes one level higher than the config.

Reply:
do not confuse overlays with the `overlay` argument to capacitate.
overlays are a bit tricky because imported flakes may depend on their overlays, but its difficult/dangerous to both not applying them and forcing the importing flake to explicitly add those overlays or appling them in the right order (given such an order exists) where they are above the package conflict resolution.

Also, do we apply overlays before the floxpkgs defined ones or after?

t;dr; overlays are tricky...
I'm rather thinking about adding a function `withOverlay` to the context or nixpkgs, so overlays are necessarily applied locally only rather than flake wide.

Reply:
Oh an you also want this to apply for build time as well, not just in an environment
e.g. if you have an unfree library you depend on...